### ISSUE TITLE
Black Screen

### ISSUE THREAD

**Issue by lmbz115** [bigtreetech/BIGTREETECH-TFT35-V3.0#148]
Hey! I apologize in advance I work and may not reply to this thread right away.

I have a BTT TFT35 v3.0 and SKR V1.4 TURBO. I updated marlin to the latest version the way I normally do by throwing it into the USB mode and directly compiling it into my onboard micro. However, since the update it seemed to have bricked my screen where I only receive a black screen where the backlight appears to be on. I completely disassembled the printer to just the board (steppers removed) and the screen that's connected to just the RS232 black 5 pin connector (I am certain that it is connected properly because I can use the reset button still, the red and green LED from when the SD card is inserted are powered, and occasionally I can hold the knob down and hear a clicking sound).

I've attempted to reflash a recompiled fresh TFT firmware from the main firmware branch to no resolve. I attempted to use the reset.txt file method to try to hard reset it but even then the calibration screen never comes up. I read on an old issue here that butchering an old usb cord to only power the screen to insert the txt file may work but it did not for me (yes, I did it correctly, the red LED and green one in the back of the screen powered).

Bricked screen or is there saving me $60?

**Comment by radek8** [comment]
I have a few questions:
1. Are you saying the screen is still black?
2. Is it possible to switch between simulated Merlin mode and touch mode?
3. Are at least the icons in touch mode displayed?
4. If you insert a new TFT FW on the SD card and reset. Will the bin file be renamed?
5. Can you insert photos of the display and cable connections here?

**Comment by lmbz115** [comment]
Through it all, when there’s power to the screen it simply doesn’t display anything. I cannot switch between modes as holding in the button simply just gives a clicking noise. No icons whatsoever are present and when I touch the screen nothing seems to happen. When an SD card is inserted, the green LED on the back turns on, however, it doesn’t seem to actually take within the device because the firmware isn’t updated to remove the .bin.

**Comment by lmbz115** [comment]
![716F4A7B-D16A-4FD7-9915-A0DC2B8CC0E0](https://user-images.githubusercontent.com/76456715/172667321-e9f6e231-f27c-4ebe-a3df-42cc11258aae.jpeg)
the “black” screen it may be hard to notice but there is like a backlight on and the screen is lit up. 

![47A04202-6DCC-471C-B7BE-58538800D687](https://user-images.githubusercontent.com/76456715/172667331-6dbf574b-c6e5-453a-b797-0296b86d119f.jpeg)
The back of the display, the red LED should signal power to the display and the green signals the SD card. 

![1D29587B-98D8-4DC9-B464-269474E00CA9](https://user-images.githubusercontent.com/76456715/172667337-23896fff-3b56-4906-b677-8f5b1c8c026f.jpeg)
My mobo connection. I undid the RST wire as it has no impact over display functionality and to show the orientation. This is the correct orientation as matched with the back of the board and as how I’ve always added it. I added the EXP1 and 2 ribbons just to show that there’s no functionality in the screen whatsoever.

**Comment by radek8** [comment]
Do you have cables EXP1 and 2 connected well?
According to the photo, they are crossed.
EXP1 is connected to EXP2 and vice versa.

**Comment by radek8** [comment]
Is the serial cable connected properly?
Take a picture of the cable marking on the SKR side

**Comment by radek8** [comment]
The photos do not show if it is connected correctly

**Comment by lmbz115** [comment]
> 

They’re not crossed… they are properly connected to the corresponding EXPs (the wires are a mess cuz I don’t normally use them). 




> The photos do not show if it is connected correctly
Ah apologies, are these any better?

![899FE845-D1F8-48B3-8655-E87B03E32D76](https://user-images.githubusercontent.com/76456715/172678548-f6b1a1e6-ec83-43df-abd9-7d5aca858422.jpeg)
![AF831181-6119-4249-A699-A0B751E0E2B6](https://user-images.githubusercontent.com/76456715/172678565-6d0a0a87-609e-4c37-b77f-a1b7f1da823f.jpeg)

**Comment by lickmyballzack** [comment]
![Uploading D305101C-737C-4832-8000-0F58801A6181.jpeg…]()

**Comment by lmbz115** [comment]
![6F344E31-17E6-42AD-B28A-F3BBEDCA992E](https://user-images.githubusercontent.com/76456715/172679059-4536e2c0-7a4a-4242-b932-516704ead05b.jpeg)

**Comment by lmbz115** [comment]
For clarification, the serial cable is connected to the TFT prongs on the mobo. The reset pin is all the way to the left (according to the orientation of my pic)

**Comment by radek8** [comment]
OK thank you. The connection seems to be fine.
It looks like a damaged FW in the TFT.
Upload a new FW. If you can't, use another SD card.
Try more SD cards with FAT32 format

**Comment by lmbz115** [comment]
I’ve attempted to do this. Even went as far as trying the reset.txt to no prevail

**Comment by radek8** [comment]
If you connect the display. Can you see that the backlight of the display lights up?

**Comment by radek8** [comment]
If the display backlight is OK and you have already tried all the options, you can try to load a new bootloader.

https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/tree/master/Bootloaders/TFT35%20V3.0%20Bootloader%20fix

https://github.com/bigtreetech/BIGTREETECH-TFT35-V3.0/issues/20

**Comment by lmbz115** [comment]
> If the display backlight is OK and you have already tried all the options, you can try to load a new bootloader.
> 
> https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/tree/master/Bootloaders/TFT35%20V3.0%20Bootloader%20fix
> 
> #20

Just drop these all into the SD card and boot it?

**Comment by radek8** [comment]
No.
You need to connect it to a PC via STLink and use the bootloader upload software.

https://uloz.to/file/N8qY2gSUhn3t/en-stm32cubeprog-v2-3-0-zip#
or
https://uloz.to/file/DGLoIVlMbJCZ/en-stm32cubeprog-v2-4-0-zip#!ZJAyZwR1LzDjA2EvZQp0ZJV4MQuwJxAXLxySDKcTJRIIAzEy

**Comment by radek8** [comment]
you need this HW
![image](https://user-images.githubusercontent.com/46979052/172946482-0f4ed307-2e80-4a90-8360-c00b9e309447.png)

![image](https://user-images.githubusercontent.com/46979052/172946992-5e93795f-958c-4fb5-9b6d-a4d1a691ba2f.png)

**Comment by radek8** [comment]
https://www.youtube.com/watch?v=aTEo-VW_UBY&t=10s

**Comment by lmbz115** [comment]
> No. You need to connect it to a PC via STLink and use the bootloader upload software.
> 

Yeah I don’t have this STLink. Guessing the USB port on the display would be no bueno either? I got the second half of the USB I butchered laying around and some soldering skills. Could I potentially just make one?
